codegen/render_workspace.m

```diff
@@ -1,51 +1,64 @@
-function render_workspace( work, output, embedded_flag )
+function render_workspace( work, hfname, cfname, embedded_flag )
 %RENDER_WORKSPACE Write workspace to header file.
 
-f = fopen(output, 'w');
+incFile = fopen(hfname, 'w');
+srcFile = fopen(cfname, 'w');
 
 % Add an include-guard statement
-[~, fname, ~] = fileparts(output);
+[~, fname, ~] = fileparts(hfname);
 incGuard = [upper(fname), '_H'];
-fprintf(f, '#ifndef %s\n', incGuard);
-fprintf(f, '#define %s\n\n', incGuard);
+fprintf(incFile, '#ifndef %s\n', incGuard);
+fprintf(incFile, '#define %s\n\n', incGuard);
 
 % Include types, constants and private header
-fprintf(f, '#include \"types.h\"\n');
-fprintf(f, '#include \"qdldl.h\"\n\n');
-fprintf(f, '#include \"qdldl_interface.h\"\n');
+fprintf(incFile, '#include \"types.h\"\n');
+fprintf(incFile, '#include \"qdldl.h\"\n');
```

Comment:
I think we don't need to include `qdldl.h`. I know this was here earlier, but maybe you can just remove it.